packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -15,6 +15,64 @@ import PlaceholderComponent from '../Components/placeholder-component.jsx';
 import { ReportsMeta } from '../Components/reports-meta.jsx';
 import EmptyTradeHistoryMessage from '../Components/empty-trade-history-message.jsx';
 
+const TransactionFilter = props => {
+    const [default_value, setDefaultValue] = React.useState('all');
+
+    const filter_list = [
+        {
+            text: 'All',
+            value: 'all',
+            has_tooltip: false,
```

Comment:
Since only 2 params change, can you just add the rest at one place? And do we actually need those params?